### ISSUE TITLE
TSX support?

### ISSUE THREAD

**Issue by danawoodman** [highlightjs/highlight.js#2484]
I see a closed PR for TSX support but I cannot get any highlighting to work with `tsx` language. Is this supported? If so, how does one use it?

I'm on `v9.18.1`

**Comment by yyyc514** [comment]
We support TypeScript, but not really TSX yet. It's probably waiting on the callback support that's still in PR and then that MIGHT make it much easier to support HTML snippet in both JS and TS.

Closing as duplicate of https://github.com/highlightjs/highlight.js/issues/1155.
